_src/_posts/2018-12-19-jplotlib-help-your-teams-to-leverage-data-by-using-the-perfect-graphs.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title:  "How to help your teams to leverage data by using the perfect graphs."
+date:   2018-12-19
+tag:    technology
+categories:
+    - data-science
+    - technology
+    - python
+keywords: "python, matplotlib, visualization, jplotlib"
+author: felipegonzalez
+---
+
+<!--excerpt.start-->
+
+In these times, it is all about Data, yes..
+
+...BUT, it’s not only about having your data available in a clean and easy data-set or feeding a machine learning algorithm, it is also about reading it, sharing it so other people (more or less technical than you) can understand the trends without too many explanations or even better, without feeling that the chart is so over-complicated that they can not even try it.
+
+Because of that, at Jampp, we create our own Matplotlib wrapper, JPlotlib.
+So everyone can use an already customized, Jampp branded and easy to go Python library to introduce data results with no effort.
+
+<!--excerpt.end-->
+
+Before, with an open source library defaults...
+![ open source library default histogram ]({{site.url}}/assets/images/jplotlib/old_histogram.png){: .center-image }
+
+After, with JPlotLib:
+![ jplotlib histogram ]({{site.url}}/assets/images/jplotlib/histogram.png){: .center-image }
+
+You might be thinking: Why? Some libraries already have what you need.
```

Comment:
Maybe you can add quotes or make it italic, so it's more clear that both sentences are a single quote.